js/build.js

```diff
@@ -596,8 +595,8 @@ Fliplet.Widget.instance('chat', function (data) {
 
           // Check if time group is empty, if it is, remove it
           $('.chat-group-holder').each(function() {
-            if ( !$.trim( $(_this).html() ).length ){
-              $(_this).parents('.chat-users-group').remove();
+            if ( !$.trim( $(this).html() ).length ){
```

Comment:
is it fine that `this` is not the same scope as the previous `_this`? I'm asking because inside the jQuery function the scope is different.

Reply:
@squallstar yes, it was a mistake. This caused component to break